src/server/pfs/server/pfsserver.proto

```diff
@@ -4,13 +4,11 @@ package pfsserver;
 option go_package = "github.com/pachyderm/pachyderm/v2/src/server/pfs/server";
 
 message ShardTask {
-  int64 index = 1;
-  repeated string inputs = 2;
+  repeated string inputs = 1;
```

Comment:
It's probably safer to not renumber these, but I guess if we clear etcd, and the upgrade happens on all the workers and pachds before any work is scheduled, then it shouldn't be a problem.

Reply:
We shouldn't support compatibility with prior etcd state. It would be a bug if we used any of it anyways. The model we should have is that all of the etcd state is deleted during an upgrade and we should be able to make whatever changes we want to the format / what we store in etcd.